### PR TITLE
Fix defferables constraints with schemas

### PR DESCRIPTION
**Pull request by webaaz**
Fix defferable constraints with schema escaping  'schema.constraint' in '"schema"."constraint"' instead of '"schema.constraint"'

### CODE REVIEW

**Comment by stood** [comment]
Add the test ?

File: sources/lib/ModelLayer/ModelLayer.php [hunk 1]
```diff
@@ -92,7 +92,16 @@ protected function setDeferrable(array $keys, $state)
         } else {
             $string = join(
                 ', ',
-                array_map(function ($key) { return $this->escapeIdentifier($key); }, $keys)
+                array_map(
+                    function ($key) {
+                        $parts = explode('.', $key);
+                        foreach ($parts as $part) {
+                            $escapedParts[] = $this->escapeIdentifier($part);
```
Comment (chanmix51) [inline sources/lib/ModelLayer/ModelLayer.php]: Coding convention: Pomm uses underscore format for variables (PSR2 compatible).

Comment (webaaz) [inline sources/lib/ModelLayer/ModelLayer.php]: it's just because it's you

**Comment by webaaz** [comment]
Oops, there's my first unit test too, but it's not tested yet!

**Comment by stood** [comment]
Yes because with atoum you doesn't can test a protected function

**Comment by chanmix51** [comment]
Merged in e45785e.

**Pull request merged**